frontend/src/scenes/surveys/surveyLogic.tsx

```diff
@@ -269,10 +269,10 @@ export const surveyLogic = kea<surveyLogicType>([
     forms(({ actions, props, values }) => ({
         survey: {
             defaults: { ...NEW_SURVEY } as NewSurvey | Survey,
-            errors: ({ name, questions }) => ({
+            errors: ({ name, questions, type }) => ({
                 name: !name && 'Please enter a name.',
                 questions: questions.map((question) => ({
-                    question: !question.question && 'Please enter a question.',
+                    question: type !== SurveyType.API && !question.question && 'Please enter a question.',
```

Comment:
I think we have different ideas for what the headless mode does - let's chat during stand up!